rust-toolchain

```diff
@@ -1,5 +1,5 @@
 [toolchain]
 channel = "nightly-2022-09-24"
-components = ["clippy", "rustfmt"]
+components = ["clippy", "rustfmt", "llvm-tools-preview"]
```

Comment:
What's this for?

Reply:
It's for the test code coverage.

Reply:
https://github.com/zeitgeistpm/zeitgeist/blob/63d5fda89e000e6a75ccc5771860710cd859b07c/.github/workflows/rust.yml#L71-L100

Does this help?

Reply:
https://blog.rng0.io/how-to-do-code-coverage-in-rust

Reply:
Ah, yeah, thanks, I just can't read.

Reply:
https://github.com/zeitgeistpm/zeitgeist/blob/1f7ea6192eb9ad5921a99ad959e14d9065d8ca09/scripts/tests/aux-functions.sh#L20